hamlpy/elements.py

```diff
@@ -1,6 +1,8 @@
+from __future__ import print_function, unicode_literals
+
 import re
+import six
```

Comment:
Six seems to be missing from `setup.py` and from the `requirements` file.

This apparently come from the fact that `mock` pulls it for testing.